CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+
+
+## v0.7.0 (TBD)
+
+Starting a Changelog.
+
+### New features and changes
+
+* Dropped Python 2 support. Python >= 3.6 is now required.
+* Added new LicenseListVersion tag.
+* Added `pyspdxtools_convertor` and `pyspdxtools_parser` CLI scripts. See [the readme](README.md) for usage instructions.
+* Updated the tools to support SPDX versions up to 2.3 and to conform with the specification. Apart from many bugfixes 
+  and new properties, some of the more significant changes include:
+    * Support for multiple packages per document
+    * Support for multiple checksums for packages and files
+    * Support for files outside a package
+    * Removed example documents from production code. Added additional up-to-date examples to test files.
+* **Note**: Validation was updated to follow the 2.3 specification. Since there is currently no support for 
+  version-specific handling, some details may be handled incorrectly for documents using lower
+  versions. The changes are mostly restricted to properties becoming optional and new property values becoming
+  available, and should be of limited impact. See https://spdx.github.io/spdx-spec/v2.3/diffs-from-previous-editions/
+  for a list of changes between the versions.
+* Pytest was introduced as the preferred test framework
+* Improved error message handling and display
+* Extended the contribution guidelines
+* Improved tag/value output structure
+* Added .editorconfig and pyproject.toml
+* Improved handling of JSON-specific properties `documentDescribes` and `hasFiles`
+* Fixed annotation handling for the JSON and Tag/Value formats
+* Free form text values in Tag/Value no longer require `<text>` tags if they don't span multiple lines
+
+### Contributors
+
+Thank you to these contributors:
```

Comment:
This sounds weird to me. Maybe something like "This release was made possible by the following contributors, thank you very much!" or something...?